package.json

```diff
@@ -65,5 +65,8 @@
     "tough-cookie": "2.3.2",
     "url-search-params": "0.6.1",
     "uuid": "3.0.1"
+  },
+  "devDependencies": {
+    "iron": "4.0.4"
```

Comment:
this is used for mocking an encrypted cookie (which is done in Hapi using Iron, but Iron isn't an exposed package)